### ISSUE TITLE
Standard notes does not work/run in clean installation of Ubuntu 22.04.2

### ISSUE THREAD

**Issue by Shmendrich** [standardnotes/forum#3090]
I freshly installed Ubuntu in a virtualbox, then installed standard-notes using the snap desktop interface, and when I try to open it I get the following error. 

I uninstalled standard notes, reinstalled it
That didnt work, so I reinstalled ubuntu, installed notes again, this tome using commandline, and the issue persists.

Any ideas would be appreciated

snap install standard-notes

```
ben@ASDUbuntu:~$ sudo snap install standard-notes

standard-notes 3.160.18 from Standard Notes (standardnotes✓) installed

ben@ASDUbuntu:~$ standard-notes

Set user data dir to /home/ben/snap/standard-notes/common

Migration: moving /home/ben/snap/standard-notes/708/.config/Standard Notes/logs to /home/ben/snap/standard-notes/common

Migration: finished moving contents to /home/ben/snap/standard-notes/common.

Error reading store file Error: ENOENT: no such file or directory, open '/home/ben/snap/standard-notes/common/user-preferences.json'

    at Object.openSync (node:fs:594:3)

    at Object.func [as openSync] (node:electron/js2c/asar_bundle:2:1822)

    at Object.readFileSync (node:fs:462:35)

    at t.readFileSync (node:electron/js2c/asar_bundle:2:9165)

    at q (/snap/standard-notes/708/resources/app.asar/dist/index.js:2:121907)

    at new V (/snap/standard-notes/708/resources/app.asar/dist/index.js:2:122380)

    at /snap/standard-notes/708/resources/app.asar/dist/index.js:2:248245

    at 7696 (/snap/standard-notes/708/resources/app.asar/dist/index.js:2:248774)

    at n (/snap/standard-notes/708/resources/app.asar/dist/index.js:2:766496)

    at /snap/standard-notes/708/resources/app.asar/dist/index.js:2:767007 {

  errno: -2,

  syscall: 'open',

  code: 'ENOENT',

  path: '/home/ben/snap/standard-notes/common/user-preferences.json'

}

A JavaScript error occurred in the main process

Uncaught Exception:

TypeError: Cannot read properties of undefined (reading 'startsWith')

    at /snap/standard-notes/708/resources/app.asar/dist/index.js:2:248283

    at 7696 (/snap/standard-notes/708/resources/app.asar/dist/index.js:2:248774)

    at n (/snap/standard-notes/708/resources/app.asar/dist/index.js:2:766496)

    at /snap/standard-notes/708/resources/app.asar/dist/index.js:2:767007

    at Object.<anonymous> (/snap/standard-notes/708/resources/app.asar/dist/index.js:2:767020)

    at Module._compile (node:internal/modules/cjs/loader:1141:14)

    at Module._extensions..js (node:internal/modules/cjs/loader:1196:10)

    at Module.load (node:internal/modules/cjs/loader:1011:32)

    at Module._load (node:internal/modules/cjs/loader:846:12)

    at f._load (node:electron/js2c/asar_bundle:2:13330)



(standard-notes:6815): Gtk-WARNING **: 20:27:29.828: Theme parsing error: gtk.css:1422:23: 'font-feature-settings' is not a valid property name



(standard-notes:6815): Gtk-WARNING **: 20:27:29.830: Theme parsing error: gtk.css:3308:25: 'font-feature-settings' is not a valid property name



(standard-notes:6815): Gtk-WARNING **: 20:27:29.831: Theme parsing error: gtk.css:3770:23: 'font-feature-settings' is not a valid property name

[7000:0531/202731.020825:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization

[7015:0531/202731.197562:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.

```

**Comment by effieeee** [comment]
Looks like this might be similar or related to this [initial report](https://github.com/standardnotes/forum/issues/3058), so I'll keep track of things there 🙏

**Comment by ErikKalkoken** [comment]
I have the same issue with the snap variant. Occurred with Standard Notes 3.161.1.

The appimpage variant works fine.

**Comment by amanharwara** [comment]
Should be fixed with [v3.169.26](https://github.com/standardnotes/app/releases/tag/%40standardnotes%2Fdesktop%403.169.26)

**Comment by effieeee** [comment]
Closing this for now, please feel free to reopen if you notice any other issues after the update!
